packages/elements-core/src/components/Docs/Model/Model.tsx

```diff
@@ -1,20 +1,31 @@
+import { JsonSchemaViewer } from '@stoplight/json-schema-viewer';
+import { CLASSNAMES } from '@stoplight/markdown-viewer';
+import { Heading } from '@stoplight/mosaic';
 import { withErrorBoundary } from '@stoplight/react-error-boundary';
-import { Classes } from '@stoplight/ui-kit';
 import cn from 'classnames';
 import { JSONSchema7 } from 'json-schema';
 import * as React from 'react';
 
-import { SchemaAndDescription } from '../../SchemaAndDescription';
+import { useInlineRefResolver } from '../../../context/InlineRefResolver';
+import { MarkdownViewer } from '../../MarkdownViewer';
 import { DocsComponentProps } from '..';
 
 export type ModelProps = DocsComponentProps<JSONSchema7>;
 
-const ModelComponent: React.FC<ModelProps> = ({ data, className, headless }) => {
+const ModelComponent: React.FC<ModelProps> = ({ data, className, headless, nodeTitle }) => {
+  const resolveRef = useInlineRefResolver();
+  const title = data.title ?? nodeTitle;
   return (
-    <div className={cn('Model MarkdownViewer', className)}>
-      {!headless && data.title !== void 0 && <h1 className={Classes.HEADING}>{data.title}</h1>}
+    <div className={cn('Model', className)}>
+      {!headless && title !== undefined && (
+        <Heading size={1} mb={4} fontWeight="semibold">
+          {title}
+        </Heading>
+      )}
 
-      <SchemaAndDescription schema={data} description={data.description} />
```

Comment:
Perhaps we can remove whole `SchemaAndDescription` as we're not using it here anymore? There's one usage in `CodeComponent` but that could be easily replaced I guess. 
And the `SchemaTitle` from there looks quite archaic.

Reply:
I was considering that actually. Didn't know why I didn't do it, I'll inline it into `CodeComponent` (and remove redundant parts.)